subgraph/subgraph.yaml

```diff
@@ -63,31 +63,44 @@ templates:
       eventHandlers:
         - event: LockConfig(uint256,uint256,uint256)
           handler: handleLockConfig
+          receipt: true
         - event: CancelKey(indexed uint256,indexed address,indexed address,uint256)
           handler: handleCancelKey
+          receipt: true
         - event: ExpirationChanged(indexed uint256,uint256,bool)
           handler: handleExpirationChangedUntilV11
+          receipt: true
         - event: ExpirationChanged(indexed uint256,uint256,uint256,bool)
           handler: handleExpirationChanged
+          receipt: true
         - event: ExpireKey(indexed uint256)
           handler: handleExpireKey
+          receipt: true
```

Comment:
do we need it for all the events?

Reply:
I was thinking it's probably easier to add them rather than ask why something does not work in a few weeks ( I spent 20 minutes looking for why the receipt was null yesterday!)